.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "*": "prettier --write",
```

Comment:
Not sure with which file type, but I was having issue with such, can you try locally a few files like `.gitignore` or `.nvmrc` (stage them).

Reply:
You're right, that's unfortunate :/
Fixed. Wish we could efficiently reuse our scripts instead, will investigate in the future.

Reply:
Nothing is stopping us from just using them on the workflow, we don't need to use that action... We only loose some nice UI on PR.

Reply:
Yea, I mean I'll see if there's a way for us to get the best of both worlds.